src/main/java/com/aws/greengrass/mqtt/bridge/clients/LocalMqtt5Client.java

```diff
@@ -463,16 +461,10 @@ private void closeClient() {
             } finally {
                 clientStarted = false;
             }
-
```

Comment:
why aren't we closing this stuff now?

Reply:
we are, it's just happening when the client is built.  was looking at examples in crt repo and that's how they closed tls options, and worked when running uat